lib/pcb/convert-circuit-json-to-pcb-svg.ts

```diff
@@ -345,11 +343,14 @@ export function convertCircuitJsonToPcbSvg(
     }
   }
 
+  let allSvgObjects = [...unsortedSvgObjects]
   if (options?.shouldDrawRatsNest) {
     const ratsNestObjects = createSvgObjectsForRatsNest(circuitJson, ctx)
-    svgObjects = sortSvgObjectsByPcbLayer([...svgObjects, ...ratsNestObjects])
+    allSvgObjects.push(...ratsNestObjects)
   }
 
+  const svgObjects = sortSvgObjectsByPcbLayer(allSvgObjects)
```

Comment:
if you use a prefix in one place, you must _always_ use a differentiating prefix

```suggestion
  const sortedSvgObjects = sortSvgObjectsByPcbLayer(allSvgObjects)
```